src/platform/forms/tests/save-in-progress/actions.unit.spec.js

```diff
@@ -166,6 +166,21 @@ describe('Schemaform save / load actions:', () => {
           done(err);
         });
     });
+    it('calls the disability compensation api to save the form', done => {
```

Comment:
I recommend making this test name a little more generic to describe what it's really doing. For now, it's just the disability compensation API, but the _functionality_ is more like `calls the overridden API endpoint for saving forms with an override` (or something).